trieste/acquisition/function.py

```diff
@@ -1963,3 +1963,32 @@ def _get_min_value_samples(
         sampler = ExactThompsonSampler(num_samples, model, sample_min_value=True)
 
     return sampler.sample(sampled_points)
+
+
+class PredictiveVariance(SingleModelAcquisitionBuilder):
+    """
+    Builder for the determinant of the predictive covariance matrix over the batch points.
+    For a batch of size 1 it is the same as maximizing the predictive variance.
+    See also :cite:`MacKay1992` for details.
+    """
+
+    def __repr__(self) -> str:
+        """"""
+        return "PredictiveVariance()"
+
+    def prepare_acquisition_function(
+        self, dataset: Dataset, model: ProbabilisticModel, jitter: float = DEFAULTS.JITTER
+    ) -> AcquisitionFunction:
+        """
+        :param dataset: Unused.
+        :param model: The model over the specified ``dataset``.
+        :param jitter: The size of the jitter to use when stabilising the Cholesky decomposition of
+            the covariance matrix.
+        :return: The determinant of the predictive function.
+        """
+
+        def determinantcovariance(at: TensorType) -> TensorType:
+            _, covariance = model.predict_joint(at)
+            return tf.exp(tf.linalg.logdet(covariance + jitter))
+
+        return determinantcovariance
```

Comment:
I think it might be good to define `update_acquisition_function` and return `function` to avoid `prepare_acquisition_function` call

Reply:
+1 for moving out jitter argument